src/dune_file_watcher/dune_file_watcher.ml

```diff
@@ -1,25 +1,44 @@
 open! Stdune
+module Inotify_lib = Async_inotify_for_dune.Async_inotify
+
+let inotify_event_paths event =
+  match event with
+  | Async_inotify_for_dune.Async_inotify.Event.Created path
```

Comment:
You might as well use the shorter `Inotify_lib` alias here and below.

Reply:
I blame merlin-destruct. :-D